client/src/app/components/view-application/view-application.component.spec.ts

```diff
@@ -15,9 +15,9 @@ describe('ViewApplicationComponent', () => {
     });
     fixture = TestBed.createComponent(ViewApplicationComponent);
     component = fixture.componentInstance;
-    fixture.detectChanges();
```

Comment:
why did you get rid of this?

Reply:
Because I haven't set up unit tests for that file so I just wanted to make it pass